lesson_03/homework03.py

```diff
@@ -0,0 +1,149 @@
+"""This is Homework 3.1."""
+
+
+import logging
+
+# Configure logging
+logging.basicConfig(level=logging.INFO, format='')
+_log = logging.getLogger(__name__)
+# task 01 == Розділіть змінну alice_in_wonderland так,
+# щоб вона займала декілька фізичних лінії
+# task 02 == Знайдіть та відобразіть всі символи одинарної лапки (') у тексті
+# task 03 == Виведіть змінну alice_in_wonderland на друк
+alice_in_wonderland = (
+    'Would you tell me, please, which way I ought to go from here?\n'
+    'That depends a good deal on where you want to get to, said the Cat.\n'
+    "I don't much care where —— said Alice.\n"
+    "Then it doesn't matter which way you go, said the Cat.\n"
+    '—— so long as I get somewhere, Alice added as an explanation.\n'
+    "Oh, you're sure to do that, said the Cat, if you only walk long enough."
+)
+_log.info(alice_in_wonderland)
+
+"""
+    # Задачі 04 -10:
+    # Переведіть задачі з книги "Математика, 5 клас"
+    # на мову пітон і виведіть відповідь, так, щоб було
+    # зрозуміло дитині, що навчається в п'ятому класі
+"""
+# task 04
+"""
+Площа Чорного моря становить 436 402 км2, а площа Азовського
+моря становить 37 800 км2. Яку площу займають Чорне та Азов-
+ське моря разом?
+"""
+black_sea_sq = 436_402
+azov_sea_sq = 37_800
+# just need to sum two sq
+main_sq = black_sea_sq + azov_sea_sq
+_log.info(main_sq)
+
+# task 05
+"""
+Мережа супермаркетів має 3 склади, де всього розміщено
+375 291 товар. На першому та другому складах перебуває
+250 449 товарів. На другому та третьому – 222 950 товарів.
+Знайдіть кількість товарів, що розміщені на кожному складі.
+"""
+
+all_warehouses = 375_291
+# find number of all goods in warehouse N3
+warehouse3 = 375_291 - 250_449
+# find number of all goods in warehouse N1
+warehouse1 = 375_291 - 222_950
+# find number of all goods in warehouse N2
+warehouse2 = all_warehouses - (warehouse3 + warehouse1)
+_log.info(f'Warehouse N1: {warehouse1}')
+_log.info(f'Warehouse N2: {warehouse2}')
+_log.info(f'Warehouse N3: {warehouse3}')
+
+# task 06
+"""
+Михайло разом з батьками вирішили купити комп’ютер, ско-
+риставшись послугою «Оплата частинами». Відомо, що сплачу-
+вати необхідно буде півтора року по 1179 грн/місяць. Обчисліть
+вартість комп’ютера.
+"""
+
+payment_per_m = 1179
+# payment timeperiod = one year and half
+timeperiod_in_m = 18
+# pc cost = 1m payment * all m
+pc_cost = payment_per_m * timeperiod_in_m
+_log.info(pc_cost)
+
+# task 07
+"""
+Знайди остачу від діленя чисел:
+a) 8019 : 8     d) 7248 : 6
+b) 9907 : 9     e) 7128 : 5
+c) 2789 : 5     f) 19224 : 9
+"""
+
+a_value = 8019 % 8
+b_value = 9907 % 9
+c_value = 2789 % 5
+d_value = 7248 % 6
+e_value = 7128 % 5
+f_value = 19224 % 9
+_log.info(f'Results: a = {a_value} b = {b_value}')
+_log.info(f'Results: c = {c_value} d = {d_value}')
+_log.info(f'Results: e = {e_value} f = {f_value}')
```

Comment:
```suggestion
_log.info(f'Results: a = {8019 % 8} b = {9907 % 9}')
...
```
Not critical just could be simplified. We will learn f-string options and capabilities  next lesson